### PR TITLE
Frontend Page Routes in README

### PR DESCRIPTION
**Pull request by mpolimen**
For the new user page that precedes seeing their first board, I'm assuming we still make the route /home

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -1,70 +1,25 @@
...
+## Routes
 
-## Available Scripts
+`/`: Landing page
 
-In the project directory, you can run:
+`/home`: Main home page for the current cycle
 
-### `npm start`
+`/home/:id`: Main home page for any cycle with id of `id`
```
Comment (roccopolimen) [inline client/README.md]: This seems weird being "home" page, but I also don't care that much if you disagree

**Review by roccopolimen — COMMENTED** [review]
Seems doable overall

File: client/README.md (hunk 1 above)
Comment (GraceMattern) [reply client/README.md]: What if it was like dashboard instead of home?

File: client/README.md [hunk 2]
```diff
@@ -1,70 +1,25 @@
...
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+`/settings`: Settings page for the current user
 
-### `npm run build`
+`/job/:jobID`: Single job page for the job with id of `jobID` within the current cycle
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+`/job/:jobID/cycleID`: Single job page for the job with id of `jobID` within the cycle with id of `cycleID`
```
Comment (GraceMattern) [inline client/README.md]: Small comment but shouldn't it be `/job/:jobID/:cycleID" ie there should be a colon mark between the last forward slash and cycleID

Comment (mpolimen) [reply client/README.md]: Yes! Good catch

**Pull request merged**